src/ArmTemplates/Common/Templates/Backend/BackendServiceFabricProperties.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Templates.Backend
 {
-    public class BackendSubProperties
+    public class BackendServiceFabricProperties
```

Comment:
You have to update the name in the copyright comment above as well.

Reply:
Also, I noticed that a bunch of files don't have this header. Do you know if we need it at all? The project is open source. Does Microsoft have copyright even when people from outside of the org contribute?

Reply:
@Hardell I think we can resolve this for now, and I will create a separate task on removing those comments, if it is required.
What do you think?

Reply:
Agreed